cirq/google/common_serializers.py

```diff
@@ -418,6 +418,68 @@ def _convert_physical_z(op: ops.Operation, proto: v2.program_pb2.Operation):
         args=[]),
 ]
 
+
+#
+# FSim serializer
+# Only allows sqrt_iswap, its inverse, identity, and sycamore
+#
+def _can_serialize_limited_fsim(theta: float, phi: float):
+    # Identity
+    if _near_mod_2pi(theta, 0) and _near_mod_2pi(phi, 0):
+        return True
+    # sqrt ISWAP and inverse sqrt ISWAP
+    if _near_mod_2pi(abs(theta), np.pi / 4) and _near_mod_2pi(phi, 0):
+        return True
+    # Sycamore
+    if _near_mod_2pi(theta, np.pi / 2) and _near_mod_2pi(phi, np.pi / 6):
+        return True
+    return False
+
+
+LIMITED_FSIM_SERIALIZERS = [
+    op_serializer.GateOpSerializer(
+        gate_type=ops.FSimGate,
+        serialized_gate_id='fsim',
+        args=[
+            op_serializer.SerializingArg(serialized_name='theta',
+                                         serialized_type=float,
+                                         op_getter='theta'),
+            op_serializer.SerializingArg(serialized_name='phi',
+                                         serialized_type=float,
+                                         op_getter='phi')
+        ],
+        can_serialize_predicate=(lambda op: _can_serialize_limited_fsim(
+            cast(ops.FSimGate, op.gate).theta,
+            cast(ops.FSimGate, op.gate).phi))),
+    op_serializer.GateOpSerializer(
+        gate_type=ops.ISwapPowGate,
+        serialized_gate_id='fsim',
+        args=[
+            op_serializer.SerializingArg(
+                serialized_name='theta',
+                serialized_type=float,
+                op_getter=(lambda op: cast(ops.ISwapPowGate, op.gate).exponent *
+                           np.pi / 2)),
```

Comment:
Need a minus sign here, because iswap**0.5 corresponds to fsim(-pi/4, 0).

Reply:
Good catch!  I always screw up this convention.